86c71nd3q/feature/readme

Dolevg2411 opened the pull request.

Dolevg2411 committed:
Added readme to norman utils external

Dolevg2411 committed:
Removed chats marks.

(#86c72xnm)

AmitMizrachi618 committed:
Adding the licence file and updating README documentation to be short and concise.
Adding the licence file and updating README documentation to be short and concise.

(#86c71nd3q)

yonatankarimish commented on README.md:
```diff
@@ -1 +1,7 @@
+
+These utilities are used across both the Norman SDK and internal Norman services. They provide common functionality that helps developers extend capabilities, debug issues and integrate advanced features more efficiently.
+
+Comprehensive documentation is available on our website: https://sdk.norman-ai.com/utils
```
For the full reference and detailed instructions, please visit our Core SDK documentation at https://sdk.norman-ai.com/utils.

yonatankarimish commented on README.md:
```diff
@@ -1 +1,7 @@
\ No newline at end of file
+# Norman Utils Overview
+
+Norman Utils is a shared library of helper utilities designed for SDK developers working within the Norman ecosystem.
```
Norman Utils is a shared utility library hosting a selection of basic classes and tools used in the Norman ecosystem.

+ Flesh out a bit. This repo is not targeted at developers. Direct them to use the core and high-level SDKs.

yonatankarimish commented on README.md:
```diff
@@ -1 +1,7 @@
+
+Norman Utils is a shared library of helper utilities designed for SDK developers working within the Norman ecosystem.
+
+These utilities are used across both the Norman SDK and internal Norman services. They provide common functionality that helps developers extend capabilities, debug issues and integrate advanced features more efficiently.
```
Delete (not intended for direct use by developers)

yonatankarimish reviewed (changes requested):
Some wording changes. Other than that looks good.

AmitMizrachi618 committed:
CR Wording fixes to objects readme documentation. small fixes.
CR Wording fixes to objects readme documentation. small fixes.

(#86c71nd3q)

AmitMizrachi618 committed:
Wording - removing the words core-sdk, as the documentation is for everything, removing the word overview from the headers.
Wording - removing the words core-sdk, as the documentation is for everything, removing the word overview from the headers.

(#86c71nd3q)

yonatankarimish reviewed (approved):
Approved
